tests/conftest.py

```diff
@@ -40,8 +44,8 @@ def monkeysession():
 
 @pytest.fixture(scope="session")
 def lora_available() -> bool:
-    # lora does not work on cpu
-    return not vllm.config.is_cpu()
+    # lora does not work with the current set of models
+    return False
```

Comment:
I had to disable lora explicitly since I got these errors
```
Model OPTForCausalLM does not support LoRA, but LoRA is enabled

Model BloomForCausalLM does not support LoRA, but LoRA is enabled
```

Reply:
I see
```
>>> vllm.config.is_cpu()
False
```